docs/content/en/docs/user-guide/configuring-deployment/ecs.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "ECS"
+linkTitle: "ECS"
+weight: 5
+description: >
+  Specific guide for configuring Amazon ECS deployment.
+---
+
+Deploying a Amazon ECS application requires `taskdef.yaml` and `servicedef.yaml` file placing inside the application directory. Those files contain all configuration for [ECS TaskDefinition](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definitions.html) object and [ECS Service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html) object, and will be used by Piped agent while deploy your application/service to ECS cluster.
+
+If you're not familiar with ECS, you can get examples for those files from [here](/docs/examples/#ecs-applications).
+
+## Quick sync
+
+By default, when the [pipeline](/docs/user-guide/configuration-reference/#ecs-application) was not specified, PipeCD triggers a quick sync deployment for the merged pull request.
+Quick sync for a ECS deployment will roll out the new version and switch all traffic to it.
+
+## Sync with the specified pipeline
+
+The [pipeline](/docs/user-guide/configuration-reference/#ecs-application) field in the deployment configuration is used to customize the way to do the deployment.
+You can add a manual approval before routing traffic to the new version or add an analysis stage the do some smoke tests against the new version before allowing them to receive the real traffic.
+
+These are the provided stages for ECS application you can use to build your pipeline:
+
+- `ECS_CANARY_ROLLOUT`
+  - deploy workloads of the new version as CANARY variant, but it is still receiving no traffic.
+- `ECS_PRIMARY_ROLLOUT`
+  - deploy workloads of the new version as PRIMARY variant, but it is still receiving no traffic.
+- `ECS_TRAFFIC_ROUTING`
+  - routing traffic to the specified variants.
+
+and other common stages:
+- `WAIT`
+- `WAIT_APPROVAL`
+- `ANALYSIS`
+
+See the description of each stage at [Configuration Reference](/docs/user-guide/configuration-reference/#stageoptions).
+
+Here is an example that rolls out the new version gradually:
+
+``` yaml
+apiVersion: pipecd.dev/v1beta1
+kind: ECSApp
+spec:
+  pipeline:
+    stages:
+      # Deploy the workloads of CANARY variant, the number of workload
+      # for CANARY variant is equal to 10% of PRIMARY's workload.
+      # But this is still receiving no traffic.
+      - name: ECS_CANARY_ROLLOUT
+        with:
+          scale: 10
```

Comment:
nit: As commented before, this can be changed to 30.

Reply:
Addressed on https://github.com/pipe-cd/pipe/pull/2080/commits/b5f3e68d674fc950bd733abcf071688f01034a07 🙏